pandora/models/pandora.py

```diff
@@ -47,6 +47,7 @@ class PlaylistItem(PandoraModel):
     track_length = Field("trackLength", 0)
     track_token = Field("trackToken")
     audio_url = Field("audioUrl")
+    bitrate = Field("bitrate")
```

Comment:
That's what I get for merging things out of order. I added this line back and checked the rest of the patch. Looks like the rest merged successfully.